core/tests/behavior/copy.rs

```diff
@@ -40,20 +40,24 @@ pub fn behavior_copy_tests(op: &Operator) -> Vec<Trial> {
 
 /// Copy a file and test with stat.
 pub async fn test_copy_file(op: Operator) -> Result<()> {
-    let source_path = uuid::Uuid::new_v4().to_string();
-    let (source_content, _) = gen_bytes();
-
-    op.write(&source_path, source_content.clone()).await?;
+    let ascii_path_pair = (
```

Comment:
Sorry for not making my review more clear.

---

What I expect is

- Don't touch `test_copy_file`
- Add a new test case called `test_copy_file_with_non_ascii`